src/transcode.jl

```diff
@@ -121,6 +84,95 @@ function Base.transcode(codec::Codec, data::ByteData)
     throw(error[])
 end
 
+"""
```

Comment:
Thr diff is larger than necessary since you moved the documentation. Could you minimize the diff by keeping the docs where they were?